wizard/vote_on_onchain_governance.py

```diff
@@ -0,0 +1,208 @@
+import sys
+import traceback
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import List
+
+from rich import print
+
+from wizard import errors, ux
+from wizard.accounts import load_accounts
+from wizard.configuration import CONFIGURATIONS
+from wizard.constants import DEFAULT_GAS_PRICE
+from wizard.entrypoint import MyEntrypoint
+from wizard.guardians import AuthApp
+from wizard.transactions import TransactionWrapper
+
+from multiversx_sdk.core.address import Address
+from multiversx_sdk.core.transactions_factory_config import TransactionsFactoryConfig
+from multiversx_sdk.governance.governance_transactions_factory import GovernanceTransactionsFactory
+from multiversx_sdk.governance.resources import VoteType
+from multiversx_sdk.network_providers.proxy_network_provider import ProxyNetworkProvider
+
+
+def _addr_from_bech32(b: str) -> Address:
+    if hasattr(Address, "new_from_bech32"):
+        return Address.new_from_bech32(b)
+    raise errors.KnownError("This SDK requires Address.new_from_bech32.")
+
+def _vote_enum(choice: str):
+    try:
+        return {"yes": VoteType.YES, "no": VoteType.NO, "abstain": VoteType.ABSTAIN}[choice]
+    except AttributeError:
+        return {"yes": getattr(VoteType, "yes"), "no": getattr(VoteType, "no"), "abstain": getattr(VoteType, "abstain")}[choice]
```

Comment:
since we have the strings as choices for the vote this can simply be: 
```py
return {"yes": VoteType.YES, "no": VoteType.NO, "abstain": VoteType.ABSTAIN}[choice]
```

we can also rename this to something like `get_vote_type_from_arg` or something similar.